graylog2-server/src/main/java/org/graylog2/indexer/retention/strategies/ClosingRetentionStrategyConfig.java

```diff
@@ -48,4 +51,13 @@ public static ClosingRetentionStrategyConfig create(@JsonProperty("max_number_of
     public static ClosingRetentionStrategyConfig createDefault() {
         return create(DEFAULT_MAX_NUMBER_OF_INDICES);
     }
+
+    @Override
+    public void validate(ElasticsearchConfiguration elasticsearchConfiguration) throws IllegalArgumentException {
```

Comment:
do we need to do this for every retentionconfig? can we even create one if the binding is missing? :thinking:

Reply:
nevermind. let's keep it as is

Reply:
Just to prevent people from still setting wrong action types via api. Strategy itself wont even get bound  but the configuration will still be valid otherwise